PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ShadowEffectFormat.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Drawing;
+using Microsoft.Office.Core;
+using PowerPointLabs.Utils;
+using ShadowFormat = Microsoft.Office.Interop.PowerPoint.ShadowFormat;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+
+namespace PowerPointLabs.SyncLab.ObjectFormats
+{
+    public class ShadowEffectFormat: Format
+    {
+        public override bool CanCopy(Shape formatShape)
+        {
+            return formatShape.Shadow.Visible.Equals(MsoTriState.msoTrue);
+        }
+
+        public override void SyncFormat(Shape formatShape, Shape newShape)
+        {
+            ShadowFormat srcFormat = formatShape.Shadow;
+            ShadowFormat destFormat = newShape.Shadow;
+            // the order in which the items are applied is extremely important
+            // Type seems to change other items like Offset once it is set
+            // A more through investigation is required to learn the exact effects
+            destFormat.Visible = srcFormat.Visible;
+            destFormat.Type = srcFormat.Type;
+            destFormat.Style = srcFormat.Style;
+            destFormat.ForeColor = srcFormat.ForeColor;
+            destFormat.Obscured = srcFormat.Obscured;
+            destFormat.RotateWithShape = srcFormat.RotateWithShape;
+            destFormat.Blur = srcFormat.Blur;
+            destFormat.OffsetX = srcFormat.OffsetX;
+            destFormat.OffsetY = srcFormat.OffsetY;
+            destFormat.Transparency = srcFormat.Transparency;
+        }
+
+        public override Bitmap DisplayImage(Shape formatShape)
+        {
+            // change transparency to make shadow more visible
```

Comment:
This is to make it more visible on the SyncLab Pane?

Reply:
Oh just saw your github comment. Hmm I'm not sure if you need to write it clearer in the comment that it is **only** on the SyncLab Pane.